src/AITS/PersonLookupConfidential.php

```diff
@@ -171,6 +176,16 @@ public function getEmail(): string
         return $this->email;
     }
 
+    public function getEmployee(): array
+    {
+        return $this->employee;
+    }
+
+    public function getOrganizationDescription(): string
+    {
+        return $this->organizationDescription;
+    }
```

Comment:
@hualiu6 , @jjonesuic - see my previous comment. I don't think we should include this getter. I'm open to hearing counter arguments for keeping it though.

Reply:
@thedanpaz @jjonesuic Agree with this, was thinking this way too, but thought this is more straight forward, since student may not have a unit, we do not need to provide this function.